src/Theme_Command.php

```diff
@@ -716,6 +716,37 @@ public function is_installed( $args, $assoc_args = array() ) {
 		}
 	}
 
+	/**
+	 * Checks if a given theme is active.
+	 *
+	 * Returns exit code 0 when active, 1 when not active.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <theme>
+	 * : The plugin to check.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Check whether theme is Active; exit status 0 if active, otherwise 1
+	 *     $ wp theme is-active twentyfifteen
+	 *     $ echo $?
+	 *     1
+	 *
+	 * @subcommand is-active
+	 */
+	public function is_active( $args, $assoc_args = array() ) {
+		$network_wide = \WP_CLI\Utils\get_flag_value( $assoc_args, 'network' );
```

Comment:
Unused variable right here.

Reply:
Yeap, you're right. Gonna get rid of it. Thanks!